### PR TITLE
Allow  logout

### PR DESCRIPTION
**Pull request by webervin**
Simple implementation of https://github.com/AndyClausen/yarn-plugin-gcp-auth/issues/6

### CODE REVIEW

**Comment by AndyClausen** [comment]
Could you document this in the readme as well?

**Comment by webervin** [comment]
I ammended commit to include some README changes  ~but seems force pushing to my branch does not play nicelly with PR. Should I create new PR instead now?~

File: README.md [hunk 1]
```diff
@@ -39,6 +39,7 @@ unsafeHttpWhitelist:
 ## Commands
 
 - `yarn gcp-auth refresh`: clears plugin cache and forces the plugin to fetch a new token.
+- `yarn gcp-auth logout`: clears plugin cache, helps ensured no token left in the system.
```
Comment (AndyClausen) [inline README.md]:
```suggestion
- `yarn gcp-auth logout`: clears plugin cache to ensure no token is left in the system.
```

**Review by AndyClausen — CHANGES_REQUESTED** [review]

**Review by AndyClausen — APPROVED** [review]
Sorry for the wait, looks good :)

**Pull request merged**